test/unitTests.js

```diff
@@ -1,32 +1,52 @@
 const RecommendController = require('../travvy-server/src/controllers/RecommendController.js')
 const AuthenticationController = require('../travvy-server/src/controllers/AuthenticationController.js')
 
```

Comment:
I just moved around some of the lines of code to make the unitTests.js file more organized for each type of test